Update user-research.md

pavlenex opened the pull request:
Co-Authored-By: @sqcken 
#202

Bosch-0 commented on guide/designing-products/user-research.md:
```diff
@@ -84,11 +83,12 @@ There are many ways in which research and development are similar to other digit
+When designers begin to develop a product, they often start with a specific country, culture, language, or legal framework in mind. This is especially true for financial applications, which are highly regulated and need to evolve slowly and strategically. While Bitcoin was originally developed for an English speaking audience, it didn’t take long for its intended use case to naturally proliferate ...
 
-This has resulted in diverse and lively online communities that are deeply invested in Bitcoin. Designers and researchers have vast amounts of public conversation at their hands that help them better understand user needs and issues. These communities are also great for direct interaction to brainstorm ideas, get feedback on potential solutions, and get users involved in the design process.
+Online communities rapidly formed and multiplied around Bitcoin. Vast amounts of public conversation helped designers and developers comprehend what was possible with Bitcoin, and what was needed from Bitcoin.
```
```suggestion
Online communities rapidly formed and multiplied around Bitcoin. Vast amounts of public conversation helped designers and developers comprehend what was possible, and what was needed from Bitcoin.
```

Bosch-0 commented on guide/designing-products/user-research.md:
```diff
@@ -84,11 +83,12 @@ There are many ways in which research and development are similar to other digit
-Some of the following links have been around for a long time and provide a huge archive and history of information and conversation around bitcoin:
 - [Bitcoin on Github](https://github.com/bitcoin)
 - [Bitcoin mailing list](https://lists.linuxfoundation.org/mailman/listinfo/bitcoin-dev)
 - [Bitcointalk](https://bitcointalk.org/)
```
```suggestion
- [Bitcointalk](https://bitcointalk.org/)
- [Bitcoin subreddit](https://www.reddit.com/r/Bitcoin/)
- [Bitcoin Core dev meeting logs](http://www.erisian.com.au/meetbot/bitcoin-core-dev/)
```

Two suggestions for some archives which have a great deal of resources hidden within.

Bosch-0 commented on guide/designing-products/user-research.md:
```diff
@@ -98,7 +98,7 @@ Some of the following links have been around for a long time and provide a huge
 ### An unknowable userbase
 
-The Bitcoin protocol works with addresses and has otherwise no sense of identity that allows us to find out who uses the system and performs various functions. Analyzing the blockchain allows for some basic transaction behavior analysis, but analysis cannot tie it to specific persons or locations. While this is generally a fantastic user benefit for privacy and security, it is harder to understan ...
+The Bitcoin protocol only knows its users through pseudonymous addresses. We can analyze the Bitcoin blockchain for fundamental transaction behavior analysis, but further analysis of addresses and transactions can be complicated. While this provides incredible privacy and security for users, it can make things challenging for user researchers.
```
```suggestion
Bitcoin only knows its users through pseudonymous addresses. We can analyze on-chain metrics for things like fundamental transaction behaviors, but analysis of addresses and transactions beyond this can be complicated. While Bitcoin's privacy provides is a fundamental aspect, it can make things challenging for user researchers.
```

Bosch-0 commented on guide/designing-products/user-research.md:
```diff
@@ -113,7 +113,7 @@ The Bitcoin protocol works with addresses and has otherwise no sense of identity
 %}
 
-While we cannot know who is directly responsible for the activity, we can analyze the overall network activity. No other currency in history has had a complete and accurate database of all transactions. What can we learn from this data that allows us to improve the system for users?
+While we often can’t know who is responsible for specific single transactions, we can still analyze overall network activity. No other currency in history has offered such a complete and accurate database of transactions. Here’s how we can use this data to improve bitcoin for users:
```
```suggestion
While we often can’t know who is responsible for specific single transactions, we can still analyze overall network activity. No other currency in history has offered such a complete and accurate ledger of transactions. Here’s how we can use this data to improve Bitcoin for users:
```

Bosch-0 replied:
There isn't examples listed after the last statement which implies there is some.

Bosch-0 commented on guide/designing-products/user-research.md:
```diff
@@ -130,11 +130,11 @@ While we cannot know who is directly responsible for the activity, we can analyz
+New technologies can take decades to reach mainstream adoption. Bitcoin is, in our opinion, still on its way up the adoption curve, which means it and we are early. This affects research in multiple ways. Early adopters have different use cases and needs than users who adopt Bitcoin ten years from now. Products designed today could be outdated by the time that the early majority embraces Bitcoin.
 
-Secondly, as the technology is maturing, its capabilities evolve as well. Scaling and performance improvements make complex transactions more affordable, better privacy, and security properties allow individuals to [self-custody](/guide/designing-products/principles/#self-custody) more comfortably and much more. There is an interplay between the size of the user base and the maturity of the techn ...
+Secondly, as Bitcoin matures, its capabilities will evolve. Scaling and performance improvements will make complex transactions more affordable, private, and secure. These properties will make early and later adopters more comfortable embracing bitcoin both as a store of value and as a medium of exchanges.
```
```suggestion
Secondly, as Bitcoin matures, its capabilities will evolve. Scaling and performance improvements will make transactions more affordable, private, and secure. These improvements will make people more comfortable embracing bitcoin as both a store of value and medium of exchange.
```

Bosch-0 reviewed (changes requested):
LGTM, lots of great simplifications, left some comments.

GBKS commented:
Again, some rationale for the changes would be great. I don't think we should get in the habit of just posting PRs without proper descriptions. As is, I have to guess why these changes were made and have no insight into why the author thinks this PR makes the text better in their opinion.

GBKS commented on guide/designing-products/user-research.md:
```diff
@@ -42,9 +42,9 @@ Illustration sources
 # User research
 
-We design digital products for the benefit of others. To do that, we need to understand who those others are and their unique needs, desires, and behaviors. A comprehensive design strategy balances user needs with considerations around the technology, business models, societal and economic impact, and other factors. In this section, we focus on understanding the users better.
+We design products to benefit users. That’s why it is necessary to understand their needs, desires, and behaviors.
```
I think this removes the important point that design is not solely about users. It is often about balancing user needs with those other aspects (tech, business model, etc). Bitcoin is a prime example, as the tech is highly opinionated and forces users to change certain behaviors.

pavlenex replied:
Fixed in https://github.com/BitcoinDesign/Guide/pull/365/commits/7a005dfb2b53e00b399f33ff33963a3f2a505a3d

GBKS commented on guide/designing-products/user-research.md:
```diff
@@ -57,19 +57,18 @@ We design digital products for the benefit of others. To do that, we need to und
+Prioritizing features that make the most significant difference is crucial. Those features are seldom apparent, and their discovery is often the result of debate and critical thinking. If you find yourself stuck, try making data-informed decisions.
 
-User research creates data from the start and end points of our efforts. Products are created because there are specific unaddressed needs, then they go through development and finally back in the hands of the target audience for use and evaluation. We are often too occupied with the complexities of development, but the real measure of success is whether users' needs are satisfied.
+The results are appropriate product roadmaps, better-focused team efforts, and happier users. This is especially important for projects with limited resources, in which many open-source efforts are.
```
Removes a point I was trying to explicitly make about being too focused on the tech while not really addressing user needs. It was meant to make the reader question whether this is something they are "guilty of".

pavlenex replied:
Reverted in https://github.com/BitcoinDesign/Guide/pull/365/commits/7a005dfb2b53e00b399f33ff33963a3f2a505a3d

GBKS commented on guide/designing-products/user-research.md:
```diff
@@ -57,19 +57,18 @@ We design digital products for the benefit of others. To do that, we need to und
+Prioritizing features that make the most significant difference is crucial. Those features are seldom apparent, and their discovery is often the result of debate and critical thinking. If you find yourself stuck, try making data-informed decisions.
 
-User research creates data from the start and end points of our efforts. Products are created because there are specific unaddressed needs, then they go through development and finally back in the hands of the target audience for use and evaluation. We are often too occupied with the complexities of development, but the real measure of success is whether users' needs are satisfied.
+The results are appropriate product roadmaps, better-focused team efforts, and happier users. This is especially important for projects with limited resources, in which many open-source efforts are.
```
```suggestion
The results are appropriate product roadmaps, better-focused team efforts, and happier users. This is especially important for projects with limited resources, which many open-source efforts are.
```

pavlenex replied:
I don't think this sounds grammatically correct.

GBKS commented on guide/designing-products/user-research.md:
```diff
@@ -84,21 +83,24 @@ There are many ways in which research and development are similar to other digit
 %}
 
-Most digital products start with a specific country, culture, language or legal framework in mind. This is especially true for financial applications, which are typically highly regulated and need to slowly and strategically evolve. While Bitcoin did start with a single language, its use case was not tied to any particular context, and this has resulted in a much more diverse user base from the s ...
+When designers begin to develop a product, they often start with a specific country, culture, language, or legal framework in mind. This is especially true for financial applications, which are highly regulated and need to evolve slowly and strategically. While Bitcoin was originally developed for an English speaking audience, it didn’t take long for its intended use case to naturally proliferate ...
```
Are you sure you meant to use `proliferate` here?

pavlenex replied:
Reproduce rapidly, spread, so yeah it makes sense to me, or do you have another better-suited word in mind?

GBKS replied:
It just changes the original point. This new version states that Bitcoin got quickly adopted by english-speaking audiences because it was first released in English. The original point was that Bitcoin is not limited by context (local laws, culture, borders...), which allowed it to have a diverse user base right away (meaning people all over the world could use it) - despite the original limitation of being English-only. That builds on the argument of this paragraph that researchers need to think differently about their users - global and diverse from the start vs start local and become global. Does that make sense?

Bosch-0 replied:
Agree with @GBKS, doesn't read as clearly or tie in as nicely with the rest of the paragraph.

pavlenex replied:
@GBKS @Bosch-0 Reverted in https://github.com/BitcoinDesign/Guide/pull/365/commits/66c0c6f3821190af411958971a2342857cfa0f3a

GBKS commented on guide/designing-products/user-research.md:
```diff
@@ -84,21 +83,24 @@ There are many ways in which research and development are similar to other digit
+When designers begin to develop a product, they often start with a specific country, culture, language, or legal framework in mind. This is especially true for financial applications, which are highly regulated and need to evolve slowly and strategically. While Bitcoin was originally developed for an English speaking audience, it didn’t take long for its intended use case to naturally proliferate ...
 
-This has resulted in diverse and lively online communities that are deeply invested in Bitcoin. Designers and researchers have vast amounts of public conversation at their hands that help them better understand user needs and issues. These communities are also great for direct interaction to brainstorm ideas, get feedback on potential solutions, and get users involved in the design process.
+Online communities rapidly formed and multiplied around Bitcoin. Vast amounts of public conversation helped designers and developers comprehend what was possible, and what was needed from Bitcoin.
```
Not sure why this is written in the past now. These communities are still active and super helpful for user research today.

pavlenex replied:
Good point.

pavlenex replied:
@GBKS 

> Some of these links have been around for a while. The vast archives that they lead to still provide enormous value to anyone who wants to build Bitcoin products.


The later paragraph though explains that these are still available, what do you suggest we do here?

GBKS replied:
> These communities are also great for direct interaction to brainstorm ideas, get feedback on potential solutions, and get users involved in the design process.

I think we should keep the point being made here. Lots of designers don't realize that a great source of user feedback is so easily available to them. They look for formal research documents, when all they need to do is to tap into the online communities that are just a click away. It's something that I'd like to stress further in future discussions of user research.

Let me try to combine the brevity of the rewrite with the points I think we should make.

> Diverse online communities rapidly formed around Bitcoin. Vast amounts of public conversation can help designers and developers comprehend what is possible, and what is needed from Bitcoin. These communities are great for brainstorming ideas, getting feedback, and involving users actively in the design process.

And then I'd remove the next sentence, which is this one:

> Some of these links have been around for a while. The vast archives that they lead to still provide enormous value to anyone who wants to build Bitcoin products.

pavlenex replied:
@GBKS  Addressed in https://github.com/BitcoinDesign/Guide/pull/365/commits/66c0c6f3821190af411958971a2342857cfa0f3a

GBKS commented on guide/designing-products/user-research.md:
```diff
@@ -84,21 +83,24 @@ There are many ways in which research and development are similar to other digit
 - [Bitcoin on Github](https://github.com/bitcoin)
 - [Bitcoin mailing list](https://lists.linuxfoundation.org/mailman/listinfo/bitcoin-dev)
 - [Bitcointalk](https://bitcointalk.org/)
+- [Bitcoin subreddit](https://www.reddit.com/r/Bitcoin/)
```
Subreddit is listed twice.

pavlenex replied:
```suggestion
```

GBKS commented on guide/designing-products/user-research.md:
```diff
@@ -84,21 +83,24 @@ There are many ways in which research and development are similar to other digit
 ### An unknowable userbase
 
-The Bitcoin protocol works with addresses and has otherwise no sense of identity that allows us to find out who uses the system and performs various functions. Analyzing the blockchain allows for some basic transaction behavior analysis, but analysis cannot tie it to specific persons or locations. While this is generally a fantastic user benefit for privacy and security, it is harder to understan ...
+Bitcoin only knows its users through pseudonymous addresses. We can analyze on-chain metrics for things like fundamental transaction behaviors, but analysis of addresses and transactions beyond this can be complicated. While Bitcoin's privacy provides is a fundamental aspect, it can make things challenging for user researchers.
```
```suggestion
Bitcoin only knows its users through pseudonymous addresses. We can analyze on-chain metrics for things like fundamental transaction behaviors, but analysis of addresses and transactions beyond this can be complicated. While the privacy Bitcoin provides is a fundamental aspect, it can make things challenging for user researchers.
```
Fixed the sentence. Still reads a bit funny though.

GBKS commented on guide/designing-products/user-research.md:
```diff
@@ -113,7 +115,7 @@ The Bitcoin protocol works with addresses and has otherwise no sense of identity
 %}
 
-While we cannot know who is directly responsible for the activity, we can analyze the overall network activity. No other currency in history has had a complete and accurate database of all transactions. What can we learn from this data that allows us to improve the system for users?
+While we often can’t know who is responsible for specific single transactions, we can still analyze overall network activity. No other currency in history has offered such a complete and accurate ledger of transactions. Here’s how we can use this data to improve Bitcoin for users:
```
> Here’s how we can use this data to improve Bitcoin for users:

This does not make sense as the following text is a new topic and not related. The original sentence was a question meant to inspire the reader to think about new ways to use the public ledger data to create better experiences.

pavlenex replied:
What do you suggest we do with it, remove it or?

pavlenex replied:
Found a good middle ground in https://github.com/BitcoinDesign/Guide/pull/365/commits/7a005dfb2b53e00b399f33ff33963a3f2a505a3d

GBKS replied:
How about this? Makes it a little more clear that it's an appeal to the reader.

> As designers, what can we learn from this data that allows us to improve the experience of our users?

pavlenex replied:
Added in https://github.com/BitcoinDesign/Guide/pull/365/commits/66c0c6f3821190af411958971a2342857cfa0f3a not sure if that's what you meant.

GBKS commented on guide/designing-products/user-research.md:
```diff
@@ -171,7 +173,7 @@ Links to:
 %}
 
-User research should not be a one-sided extraction of information but a collaboration between researchers and users to find better solutions. Since researchers initiate research efforts, it is up to them to create frameworks that respect users' privacy. That means full upfront disclosure, open collaboration, and a mutual understanding of how research results will be used.
+User research should not be a one-sided extraction of collaboration between researchers and users. Since researchers initiate research efforts, it is up to them to create frameworks that respect users’ privacy. That means upfront disclosures, open collaboration, and transparency about how research results are used.
```
```suggestion
User research should not be a one-sided extraction of information, but a collaboration between researchers and users. Since researchers initiate research efforts, it is up to them to create frameworks that respect users’ privacy. That means upfront disclosures, open collaboration, and transparency about how research results are used.
```
Fixed broken sentence.

pavlenex commented:
> Again, some rationale for the changes would be great. I don't think we should get in the habit of just posting PRs without proper descriptions. As is, I have to guess why these changes were made and have no insight into why the author thinks this PR makes the text better in their opinion.

The majority of changes are for _conciseness_ as it's people's habit to want content instantly, nobody reads anymore, people scan through, so the goal of this and many other PR's is to make content a bit more concise and to the point.

GBKS commented:
> nobody reads anymore

I've heard this many times, and I think it's accurate in certain contexts. For example browsing news is a lot about checking headlines and digging deeper when you found a headline you found interesting, or just pattern recognition (saw 10 mentions of Bitcoin Miami -> must be important). Social media, with its infinite feeds, often also has a low signal-to-noise ratio which leads to scanning. I don't think it's accurate in other places where people intentionally seek out information, and it's also a bit dismissive of the audience. When you really care about designing something cool (or its your job), you'll read a few more sentences. As long as they are informative and save you time or energy, you'll do it. So I think our goal should be to write as concisely and clearly as we can, using both text and visuals effectively, so a ton of people can easily understand what we're trying to say. If that takes an extra sentence somewhere, we should add it. Don't you also read books like `Working in Public` from start to finish and appreciate that some areas go into more depth to clearly make certain points? 

So yeah, the 1 millionth content marketing post with 10 tips for boosting your customer satisfaction score can be cut to almost nothing because the point is primarily to get attention and not to actual inform, and readers can tell that right away and just scan the headline and move on. But I don't think that's what we're doing, and our reasons for writing concisely should be driven be respecting their time and intellect and giving them the information that they really should know.

Sorry for the rant, I just don't like dismissive attitudes of users and think there's nuance to how people go about reading and browsing.

Making things more concise has two parts, IMHO. One is using fewer, simpler words to make the same points. Second is removing points that don't add anything. Simplifying language is always great and easy to review and approve. When arguments are removed or changed, I think that a rationale should be given. Sometimes I am not certain whether the change is meant to simplify language or change the argument being made, and I'm not sure from which perspective to review it. That's why I'm asking for a bit more explanation.

Let me know if all that makes sense.

pavlenex commented:
@GBKS What we're condensing is duplicated, ranty-like sentences, and trying to make paragraphs on point. It's normal to be overly protective of the content, but if you re-read what's been addressed here you'll see it's not chopping off things. While there were some things that were causing context to be lost, those have been addressed in https://github.com/BitcoinDesign/Guide/commit/7a005dfb2b53e00b399f33ff33963a3f2a505a3d if there's something more to fix, let me know, will happily address your concerns and as always thanks for an in-depth review on this one.

GBKS commented:
Can you give an example of `ranty-like sentences`?

pavlenex commented:
@GBKS @Bosch-0 addressed remaining concerns including reverts in https://github.com/BitcoinDesign/Guide/pull/365/commits/66c0c6f3821190af411958971a2342857cfa0f3a Let me know how that looks now.

pavlenex commented:
ping for a review beg @GBKS @Bosch-0

Bosch-0 reviewed (approved):
LGTM!

danielnordh commented:
It's been a lot of back and forth to get this PR here but I think it's good to go now.
Some thoughts for any future work that is in this vein of 'wholesale tweaking for conciseness and style':

- As much as possible, try to keep PR's small
- If change is 'stylistic', maintain sentiment and meaning of content
- If change is for 'conciseness', give rational or explain why content is being removed
- Have the original proposer in the PR discussion

GBKS reviewed (approved):
Thanks, @danielnordh and @pavlenex, for the thorough review and discussion, I am going to approve and merge now.
